### PR TITLE
Cleans up atmos unary machinery code

### PR DESCRIPTION
**Pull request by nicbn**
Old PR broke github

### CODE REVIEW

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -83,59 +83,61 @@
...
-		else // Anything else
-			occupant_overlay = image(CRYOMOBS, "generic")
+		else // Drone or other
+			occupant_overlay = image(CRYOMOBS, "aliend")
 
 		occupant_overlay.dir = SOUTH
 		occupant_overlay.pixel_y = 22
 
-		if(on && !running_bob_anim && is_operational())
+		if(on && is_operational() && !running_anim)
```
Comment (Cyberboss) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: unswap these, checking a var is much faster than calling a proc

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 2]
```diff
@@ -157,60 +159,65 @@
 		on = FALSE
 		update_icon()
 		return
+	if(!occupant)
+		return 1
```
Comment (Cyberboss) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: remove the 1

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 3]
```diff
@@ -157,60 +159,65 @@
...
+
+	var/mob/living/mob_occupant = occupant
+
+	if(mob_occupant.stat == DEAD) // We don't bother with dead people.
+		return
+
+	if(mob_occupant.health >= mob_occupant.getMaxHealth()) // Don't bother with fully healed people.
+		on = FALSE
+		update_icon()
+		playsound(get_turf(src), 'sound/machines/cryo_warning.ogg', volume) // Bug the doctors.
```
Comment (Cyberboss) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm]: playsound(src,

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
@duncathan @as334

**Pull request merged**